recursive-list-main/src/components/index.tsx

```diff
@@ -0,0 +1,4 @@
+export { default as Breadcrumbs } from "./breadcrumbs/breadcrumbs.component";
```

Comment:
check if this file extension should be .ts or .tsx

Reply:
Changed from index.tsx to index.ts